### DISCUSSION TITLE
I found an unusual file modification time in the repository.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#146502] [Repositories] [answered]
### Select Topic Area

Question

### Body

https://github.com/1q23lyc45/KitsuneMagisk/blob/kitsune/gradle/wrapper/gradle-wrapper.properties

I noticed that the modification time shows "8 days ago". I'm very confused. Starting November 28, 2024 and ending November 30, I did not make any commits to the repo. 

It only started committing on Dec. 1, and today, Dec. 7, I don't know why it shows that there was a commit on Nov. 30th.
![Screenshot_2024-12-07-06-51-35.png](https://github.com/user-attachments/assets/8108995f-5adc-45af-a620-1c1bb138b135)
![IMG_20241207_065438](https://github.com/user-attachments/assets/1b0a49dc-b0f1-4fb3-96e5-24a6a72e9f88)

Is this a Firefox bug?

**Comment by MindfulLearner** [comment]
could be a branch and eliminated try to look to the cache of the .git

**Reply by ghost** [reply]
Could you please tell me what I should do?

**Reply by MindfulLearner** [reply, marked as answer]
Go to your repo in your computer

git ls-files --cached

Try to take a look.

Im not sure 100% but lets start from there.

ARE they people you know?

**Reply by ghost** [reply]
Although I don't know the reason, I reinstalled Firefox browser today and the display returned to normal.
Thank you very much!

**Reply by MindfulLearner** [reply]
Happy for that see ya!
